backend/app/assets/javascripts/spree/backend/components/selectable_table.js

```diff
@@ -0,0 +1,16 @@
+Spree.ready(function() {
+  $('table.selectable-table').each(function() {
+    var selectableTableModel = new Backbone.Model({
+      allSelected: false,
+      selectedItems: []
+    })
+
+    new Spree.Views.Tables.SelectableTable({el: this, model: selectableTableModel});
+
+    if($(this).hasClass('return-items-table')) {
```

Comment:
I wonder if this code could be moved in a location more specific to return items. This file seems to address the generic behavior for selectable tables, so it's a bit strange seeing something very specific related only to the return items table... what do you think?

Reply:
Sorry, I just realized that @tvdeyen made the same comment some minutes ago. I'll let you handle that discussion with him